dataset_conversion/convert_bids_to_nnUnetv2.py

```diff
@@ -71,20 +75,26 @@ def convert_subject(root, subject, channel, contrast, label_suffix, path_out_ima
         copy (bool): The files in the nnUNet dataset need to be symlink of copy file (False: symlink, True: copy).
         DS_name (str): Dataset name.
         channel (int): Contrast value as integer compatible with nnUNet documentation (ex: T1 = 1, T2 = 2, FLAIR = 3).
+        is_softseg (None or list): The label file have softseg values (if value = None label are not softseg)
 
     Returns:
         list_images (list): List containing the paths of training/testing images in the nnUNetv2 format.
         list_labels (list): List containing the paths of training/testing labels in the nnUNetv2 format.
 
     """
+    if is_softseg:
+        label_start_path = os.path.join(root, 'derivatives', 'labels_softseg', subject)
```

Comment:
This is not always the case actually. We have datasets where the soft segs are still within `derivatives/labels`. @valosekj any suggestions on how we can set the labels path? maybe we could check whether `labels_softseg` folder exists and if not then take the default path

Reply:
Good point!
In the `manual_correction.py` script, we have `-path-derivatives` flag allowing us to specify the derivatives folder; see [here](https://github.com/spinalcordtoolbox/manual-correction/blob/b1cf43e6e8e325fdd80ba64c4ff3b209676d91fa/manual_correction.py#L105-L110).
We have two options here:
1. add a similar flag
2. keep the code as it is and add TODO that the user has to change the path manually. I think this solution is also acceptable since the script is intended to be a template, not a universal script.

Reply:
Thanks for the info. I take the first option with the `-path-derivatives `